src/components/user-list.html

```diff
@@ -1,24 +1,75 @@
-<link rel="import" href="./user-profile.html">
+<link rel="import" href="../../node_modules/@banno/polymer/polymer.js">
+<link rel="import" href="../../node_modules/@banno/polymer/lib/elements/dom-repeat.js">
 <link rel="import" href="./user-profile.html">
 
 <dom-module id="user-list">
-    
-    <template>
-        <user-profile></user-profile>
-        <user-profile></user-profile>
-        <user-profile></user-profile>
+  <style>
+   /* Add Styles , not taking so far???*/
+   .centered{
+     text-align: center;
+   }
+   
+   .button {
+        background-color: #87CEFA;
+        border: none;
+        color: white;
+        padding: 5px 12px;
+        text-align: center;
+        text-decoration: none;
+        display: block;
+        margin: auto;
+        font-size: 16px;
+        margin-top: 25px;
+      }
+
+      button:hover {
+        background-color: rgb(196, 226, 245);
+      }
+  </style>
+  
+  <template>
+    <div class="centered">
+      <h2 class="centered">User List:</h2>
+      <button class="button" value="Refresh Page" onClick="window.location.reload();">Generate User List</button>
+    </div>
+    <template is="dom-repeat" items="[[users]]" as="user">
+      <div>
+        <user-profile 
+          id="[[user.id]]"
+          firstname="[[user.firstName]]"
+          lastname="[[user.lastName]]"
+          phone="[[user.phone]]"
+          email="[[user.email]]">
+        </user-profile>
+      </div>
     </template>
+  </template>
+  <button on-click="getUsers">Test getAllUsers()</button>
+
+  <script type="module">
+    import { Element as PolymerElement } from '@banno/polymer/polymer-element.js';
+    import { LocalStorage } from '../api/localStorage.js';
+
+    const Storage = new LocalStorage();
 
-    <script type="module">
-        import { Element as PolymerElement } from '@banno/polymer/polymer-element.js';
-        
-        class UserList extends PolymerElement {
-            static get is() { return 'user-list'; }
-        }
+    class UserList extends PolymerElement {
+      static get is() { return 'user-list'; }
 
-        customElements.define(UserList.is, UserList);
+      static get properties() {
+        return {
+          users: {
+            value: () => Storage.getAllUsers()
+          }
+        };
+      }
 
-        export default UserList;
-    </script>
+      getUsers() {
+        Storage.getAllUsers();
+        console.log('this.users', this.users);
```

Comment:
this is cool to have around locally, but we're not going to want to commit it to `master`.